docs/document/content/reference/distsql/syntax/rdl/rule-definition/sharding/create-sharding-binding-table-rule.cn.md

```diff
@@ -25,8 +25,9 @@ tableName ::=
 - 只有分片表才能创建绑定关系
 - 一个分片表只能具有一个绑定关系
 - 添加绑定关系的分片表需要使用相同的资源，并且分片节点个数相同。例如 `ds_${0..1}.t_order_${0..1}` 与 `ds_${0..1}.t_order_item_${0..1}`
-- 添加绑定关系的分片表需要对分片键使用相同的分片算法。例如 `t_order_${order_id % 2}` 与 `t_order_item_${order_item_id % 2}` 
-- 只能存在一个绑定规则，但可包含多个绑定关系，因此无法重复执行 `CREATE SHARDING BINDING TABLE RULE` 。当绑定规则已经存在但还需要添加绑定关系时，需要使用 `ALTER SHARDING BINDING TABLE RULE` 来修改绑定规则
+- 添加绑定关系的分片表需要对分片键使用相同的分片算法。例如 `t_order_${order_id % 2}` 与 `t_order_item_${order_item_id % 2}`
+- 只能存在一个绑定规则，但可包含多个绑定关系，因此无法重复执行 `CREATE SHARDING BINDING TABLE RULE`
+  。当绑定规则已经存在但还需要添加绑定关系时，需要使用 `ALTER SHARDING BINDING TABLE RULE` 来修改绑定规则
```

Comment:
Please check here again.

Reply:
> Please check here again.

In fact, I made a line break to prevent a line of text becoming too long and less readable, and such a change does not affect the output of the markdown file.

Reply:
But the line prefixed with a full stop seems weird.

Reply:
> But the line prefixed with a full stop seems weird.

You are right, I will make adjustments